metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -879,6 +879,74 @@ def _extract_lineages(self):
         return
 
 
+@dataclass
+class SnowflakeSinkConnector:
+    connector_manifest: ConnectorManifest
+    report: KafkaConnectSourceReport
+
+    def __init__(
+        self, connector_manifest: ConnectorManifest, report: KafkaConnectSourceReport
+    ) -> None:
+        self.connector_manifest = connector_manifest
+        self.report = report
+        self._extract_lineages()
+
+    @dataclass
+    class SnowflakeParser:
+        target_platform: str
```

Comment:
`target_platform` will always be `"snowflake"`, so we can remove it. I understand, we have such redundancy in other places, we should start cleaning those up, definitely not scope of this PR :)